install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3260,8 +3260,14 @@ authentication:
           # -- SPIRE Workload Attestor kubelet verification.
           skipKubeletVerification: true
           # -- SPIRE agent tolerations configuration
+          # By default the kubernetes `node-role.kubernetes.io/control-plane` is tolerated
+          # to allow the Cilium agent on this node to connect to SPIRE.
           # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
-          tolerations: []
+          tolerations:
```

Comment:
We should also tolerate `node.kubernetes.io/not-ready` and `{{ .Values.agentNotReadyTaintKey}}`

Reply:
Added